### DISCUSSION TITLE
Life without Internet

### DISCUSSION THREAD

**Discussion by janply** [NAIST-SE/AttractingContributors#610] [General]
Could you live without Internet? how long could you stay away from it?

**Comment by ellielo** [comment]
One - two hours max
I definitely can't live without it

**Comment by McJul** [comment]
Theoretically, I can live about about a week without Internet. Exclusively when I'm full of work and home stuff. However, it's too uncomfortable. Especially, when I come home after a hard day, everything I want is to turn on a TV and relax, and WTF? It doesn't work... Such situation was last month. Thankfully, I found a solution at [https://www.firesticktricks.com/how-to-reset-firestick.html](https://www.firesticktricks.com/how-to-reset-firestick.html), but I spend the whole week without any entertainment and with upset kids.
